scaladoc/src/dotty/tools/scaladoc/tasty/NameNormalizer.scala

```diff
@@ -17,6 +17,19 @@ object NameNormalizer {
       val escaped = escapedName(constructorNormalizedName)
       escaped
     }
+    
+    def ownerNameChain: List[String] = {
+        import reflect.*
+        s match
+          case s: reflect.Symbol if s.isNoSymbol => List.empty
+          case s: reflect.Symbol if s == defn.EmptyPackageClass => List.empty
+          case s: reflect.Symbol if s == defn.RootPackage => List.empty
+          case s: reflect.Symbol if s == defn.RootClass => List.empty
+          case _ => s.owner.ownerNameChain :+ s.normalizedName
+      }
```

Comment:
I think that this closing brace is misformatted.

Also (this is just a suggestion though), I'm not a fan of using a `match` statement here, since it doesn't match on anything. It is essentially an `if` with a slightly longer syntax.

Reply:
Okay sorry, it's true that I have the reflex to do a pattern matching but for this case, it loses its exhaustiveness. Thanks a lot for the feedback!